DeveloperGame/DeveloperGame/ComputerGeekGame.cs

```diff
@@ -80,8 +80,7 @@ private void ReadScoresFromFile()
 
         private void PrintIntro()
         {
-            Console.WriteLine("\nWelcome To Computer Geek Game!");
-            Console.WriteLine("\nIn this game, your knowledge about Computer Science history is going to be tested. The test cosists of multiple choice and true or false questions. " +
+            Console.WriteLine("\nWelcome To Computer Geek Game!\nIn this game, your knowledge about Computer Science history is going to be tested. The test cosists of multiple choice and true or false questions. " +
```

Comment:
Before you had 2 new lines, now you have 1

Reply:
Good, you know how to count.